README.md

```diff
@@ -89,6 +89,12 @@ To Start a Docker Compose Instance for Testing / Debugging
     
    You can start multiple compose instances on the same project as the plugin generates a unique name for each instance.
    
+   To use the static host ports instead of the Docker dynamically assigned host ports use the following command:
+   
+    dockerComposeUp -useStaticPorts
```

Comment:
Probably should remove the '-' for consistency because skipPull and skipBuild don't have '-'